app/institutions/dashboard/-components/institutional-dashboard-wrapper/template.hbs

```diff
@@ -4,9 +4,23 @@
     >
         <div local-class='banner'>
             <img src='{{@institution.bannerUrl}}' alt='{{@institution.name}}'>
-            <div>
-                {{t 'institutions.dashboard.last_update'}}
-            </div>
+            {{#if @institution.linkToExternalReportsArchive}}
+                <div>
+                    <OsfLink
+                        data-test-link-to-reports-archive
+                        data-analytics-name='Link to archived reports'
+                        local-class='download-button'
+                        @href={{@institution.linkToExternalReportsArchive}}
```

Comment:
(ux nitpick) as an external link, seems appropriate to open in a new tab/window with `target="_blank"` (unsure if there's a request either way on that)